app/src/main/java/io/homeassistant/companion/android/sensors/BluetoothSensorManager.kt

```diff
@@ -261,6 +342,33 @@ class BluetoothSensorManager : SensorManager {
         )
     }
 
+    fun updateBeaconMonitoringSensor(context: Context) {
+        val icon = if (monitoringManager.isMonitoring()) "mdi:bluetooth" else "mdi:bluetooth-off"
+
+        var state = if (!BluetoothUtils.isOn(context)) "Bluetooth is turned off" else if (monitoringManager.isMonitoring()) "Monitoring" else "Stopped"
+
+        var attr: Map<String, Any?> = mapOf()
+        if (BluetoothUtils.isOn(context) && monitoringManager.isMonitoring()) {
+            for (beacon: IBeacon in beaconMonitoringDevice.beacons) {
+                attr += Pair(beacon.uuid, beacon.distance)
+            }
+        }
+
+        // reset the last_sent_state of the sensor so it won't skip the update of attributes
+        val sensorDao = AppDatabase.getInstance(context).sensorDao()
+        runBlocking {
+            sensorDao.updateLastSentStateAndIcon(beaconMonitor.id, "", "")
```

Comment:
This function was updated in #2734 (which has been merged), please use `null` instead of `""` to force an update of the sensor